distribution/lib/Standard/Base/0.0.0-dev/src/Data/Interval.enso

```diff
@@ -1,59 +1,82 @@
-from Standard.Base import all
+import project.Data.Numbers.Number
+from project.Data.Boolean import Boolean, False
 
-import Standard.Base.Data.Interval.Bound
+## A type representing an interval bound over any order-able type.
 
-export Standard.Base.Data.Interval.Bound
+   An order-able type is one that has a
+   [total order](https://en.wikipedia.org/wiki/Total_order) defined for it.
```

Comment:
Do we support this URL syntax?

Reply:
Not changed from what was there before.

Reply:
Ok, I thought that initially but I was unable to find it existed before. Maybe I'm just too tired today.

Still thinking if worth revising - we should settle what syntax we can use in the docs. Such links would actually be useful, but let's decide if they can be used (and then use them more widely) or not - and then we should reword this.